### PR TITLE
7.10 support

### LINKED ISSUE #368: Add support for Elastic Stack 7.10.0

### PR DESCRIPTION
**Pull request by Mpdreamz**
This PR adds support for Elastic Stack 7.10.0.

Closes #368

### CODE REVIEW

**Comment by Mpdreamz** [comment]
If integration tests pass, run

```
npm run links -- --branch master
```

and commit, before merging in.

**Comment by Mpdreamz** [comment]
The deployments I ran are all 🟢  🥳

File: build/allowedValues.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "versions": [
     "6.8.12",
+    "6.8.14",
```
Comment (russcam) [inline build/allowedValues.json:4]: I think we should remove 6.8.12

**Review by russcam — APPROVED** [review]
LGTM 👍 Just left a comment about removing an older patch version

File: build/allowedValues.json [hunk 2]
```diff
@@ -10,7 +11,8 @@
     "7.6.2",
     "7.7.1",
     "7.8.1",
-    "7.9.0"
+    "7.9.4",
```
Comment (russcam) [inline build/allowedValues.json:14]: [7.9.3 is the latest release in 7.9.x](https://www.elastic.co/downloads/past-releases/elasticsearch-7-9-3)

File: build/allowedValues.json (hunk 1 above)
Comment (russcam) [reply build/allowedValues.json:3]: done

File: build/allowedValues.json (hunk 2 above)
Comment (russcam) [reply build/allowedValues.json]: amended to 7.9.3

**Pull request merged**